node/cmd/guardiand/node.go

```diff
@@ -1573,3 +1565,26 @@ func makeChannelPair[T any](cap int) (<-chan T, chan<- T) {
 	out := make(chan T, cap)
 	return out, out
 }
+
+var zeroAddress vaa.Address
```

Comment:
Why not put this inside the function validateMessage()?

Reply:
It's at least some overhead to set it to zero. Don't think it hurts to have it global.